index.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+HK&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/reset.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+    <title>Smashing Magazine</title>
+</head>
+<body>
+    <nav class="nav-header">
+        <div class="nav-item"><h3>Smashing <br> Magazine</h3></div>
+        <div class="nav-item"><h3>Articles</h3></div>
+        <div class="nav-item"><h3>Books</h3></div>
+        <div class="nav-item"><h3>Events</h3></div>
+        <div class="nav-item"><h3>Jobs</h3></div>
+        <div class="nav-item"><h3>Me</h3></div>
+        <div class="nav-item"><h3>Topics</h3></div>
```

Comment:
I believe the navbar should contain `<ul>`, `<a>`, and `<li>` tags instead of the `<div>` tags.